scitran/data/meeg/meeg.py

```diff
@@ -287,29 +293,31 @@ class MEEGError(data.DataError):
 
 
 class MEEGReader(data.Reader):
-
     """
     Parameters
     ----------
     path : str
         path to input file
     load_data : boolean
-        indicate if a reader should attempt to immediately load all data. default False.
-
+        indicate if a reader should attempt to immediately load all data.
+        Default: False.
     """
     project_properties = project_properties
     session_properties = session_properties
     acquisition_properties = acquisition_properties
 
     domain = u'meeg'
-    filetype = u'somefiletype'   # filetype within meeg domain
+    filetype = u'FIF'   # filetype within meeg domain
     state = ['orig']             # usually an 'orig' raw file gets 'reaped'
 
-    @abc.abstractmethod
     def __init__(self, path, load_data=False):
         super(MEEGReader, self).__init__(path, load_data)
+        with tarfile.open(path) as tar:
+            for member in tar.getmembers():
+                fid, tree = fiff_open(tar.extractfile(member))[:2]
+                info = read_meas_info(fid, tree)[0]
+                print(info)
```

Comment:
Here are the lines of `mne` that extract the measurement information and print a representation of it... now I need to know what properties / dict object / etc. I need to populate, I think.